.github/workflows/happo.yml

```diff
@@ -0,0 +1,32 @@
+name: Happo CI
+
+on:
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - 'src/**'
+
+jobs:
+  happo:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Run Happo
+        run: yarn run happo-ci
+        if: github.actor != 'dependabot[bot]' && github.actor != 'dependabot-preview[bot]'
+        env:
+          HAPPO_IS_ASYNC: true
+          BASE_BRANCH: 'origin/develop'
+          HAPPO_API_KEY: ${{ secrets.HAPPO_API_KEY }}
```

Comment:
where are these secrets stored?

Reply:
oh is it because we have the Happo github app installed? I am not sure

Reply:
@suzubara yes, that's my understanding.